canned/postgres/postgres.go

```diff
@@ -0,0 +1,115 @@
+package postgres
+
+import (
+	"context"
+	"fmt"
+	"github.com/docker/go-connections/nat"
+	"github.com/pkg/errors"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	image             = "postgres"
+	tag               = "9.6.15"
```

Comment:
Don't you think that it will be better if we use some fresher versions by default? Like 12.1-alpine, for example.